authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -373,6 +426,65 @@ func getAuthInfoListForWildcard(subRevReq *authzv1.SubjectAccessReviewSpec, stor
 	return authInfoList, nil
 }
 
+func getAuthInfoListForCustomResource(subRevReq *authzv1.SubjectAccessReviewSpec, clusterType string) ([]azureutils.AuthorizationActionInfo, error) {
+	var authInfoList []azureutils.AuthorizationActionInfo
+	if subRevReq.ResourceAttributes.Verb == "*" {
+		for _, action := range getCustomResourceOperationsMap(clusterType) {
+			authInfoList = append(authInfoList, action)
+		}
+	} else {
+		action := getActionName(subRevReq.ResourceAttributes.Verb)
+		authInfoSingle, found := getCustomResourceOperationsMap(clusterType)[action]
+		if !found {
```

Comment:
This should be documented as one of the limitations that we need to vet with PM and eventually documented publicly.